django/db/backends/postgresql/base.py

```diff
@@ -141,6 +141,7 @@ class DatabaseWrapper(BaseDatabaseWrapper):
     ops_class = DatabaseOperations
     # PostgreSQL backend-specific attributes.
     _named_cursor_idx = 0
+    psycopg2_version = PSYCOPG2_VERSION
```

Comment:
Probably this could be removed because it seems unused since 3adc5f1ee6618a615db07d4a868b366a15c39f82.